### PR TITLE
Security Analytics—additional updates following 2.5 release

### PR DESCRIPTION
**Pull request by cwillum**
Signed-off-by: cwillum <cwmmoore@amazon.com>

### Description
Some updates didn't make it into 2.5. These are follow up changes and some refinements.

### Issues Resolved
Reordered some field mapping documentation to treat automatic mapping first and pending mapping second, changed language in the Alert setup section of detection creation, and other small refinements.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security-analytics/sec-analytics-config/detectors-config.md [hunk 1]
```diff
@@ -48,9 +50,16 @@ Security Analytics takes advantage of prepackaged Sigma rules for security event
...
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/default-mappings.png" alt="Field mapping example for pending mappings" width="85%">
+
+Although these automatic matches are normally dependable, it's still a good idea to review the mappings and verify that they are correct and matched as expected. If you find a mapping that doesn't appear to be accurate, you can use the dropdown list as described in the [Pending field mappings](#pending-field-mappings) section that follows to correct the field mapping.  
+
 ### Pending field mappings
 
-Once you navigate to the **Configure field mapping** page, the system attempts to automatically map fields between the two sources. Those field names that are not automatically mapped appear in the **Pending field mapping** table. In this table you can manually map rule fields to log fields, as shown in the following image.
+The field names that are not automatically mapped appear in the **Pending field mappings** table. In this table you can manually map rule fields to log fields, as shown in the following image.
```
Comment (alicejw-aws) [inline _security-analytics/sec-analytics-config/detectors-config.md:62]: add the colon
... in the following image:"

File: _security-analytics/sec-analytics-config/detectors-config.md [hunk 2]
```diff
@@ -59,35 +68,30 @@ While mapping fields, consider the following:
...
 
 After completing the mappings, select the **Next** button in the lower-right corner of the screen. The **Set up alerts** page appears and displays settings for an alert trigger.
 
 ## Step 3. Set up alerts
 
-At this stage, setting up alerts is optional for creating a new detector. Alerts can be configured at any time, including from the Findings window. This section describes the process for defining the alert conditions during creation of a detector. To see how to initiate creation of alerts from the Findings window, see [The findings list]({{site.url}}{{site.baseurl}}/security-analytics/usage/findings/#the-findings-list).
+The third step in creating a detector involves setting up alerts. Alerts are configured to create triggers that, when matched with a set of detection rule criteria, send a notification of a possible security event. You can select the criteria (rules, rule severity, and tags) in any combination to define a trigger. Once a trigger is defined, the alert setup lets you choose the channel on which to be notified and provides options for customizing a message for the notification.
```
Comment (alicejw-aws) [inline _security-analytics/sec-analytics-config/detectors-config.md:77]:
```suggestion
The third step in creating a detector involves setting up alerts. Alerts are configured to create triggers that, when matched with a set of detection rule criteria, send a notification of a possible security event. You can select the criteria such as rules, rule severity, and tags in any combination to define a trigger. Once a trigger is defined, the alert setup lets you choose the channel on which to be notified and provides options for customizing a message for the notification.
```

**Review by alicejw-aws — APPROVED** [review]
Nice work!

**Review by cwillum — COMMENTED** [review]
Thanks Alice.

File: _security-analytics/sec-analytics-config/detectors-config.md [hunk 3]
```diff
@@ -59,35 +68,30 @@ While mapping fields, consider the following:
...
+
+At least one alert condition is required before a detector can begin generating findings.
+{: .note }
 
-To skip directly to generating findings from the detector, select the **Remove alert trigger** button and then the **Next** button in the lower-right corner of the screen. Review the detector's definition and then select the **Create** button in the lower-right corner of the screen. The detector is created.
-{: .tip }
+You can also configure alerts from the Findings window. To see how to set up alerts from the Findings window, see [The findings list]({{site.url}}{{site.baseurl}}/security-analytics/usage/findings/#the-findings-list). A final option for adding additional alerts is to edit a detector and navigate to the **Alert triggers** tab, where you can edit existing alerts as well as add new ones. For details, see [Editing a detector]({{site.url}}{{site.baseurl}}security-analytics/usage/detectors/#editing-a-detector).
```
Comment (Naarcha-AWS) [inline _security-analytics/sec-analytics-config/detectors-config.md:82]: Could be wrong here, but I assume **Findings** refers to a section in analytics UI. Therefore, I bolded it here.

```suggestion
You can also configure alerts from the **Findings** window. To see how to set up alerts from the **Findings** window, see [The findings list]({{site.url}}{{site.baseurl}}/security-analytics/usage/findings/#the-findings-list). A final option for adding additional alerts is to edit a detector and navigate to the **Alert triggers** tab, where you can edit existing alerts as well as add new ones. For details, see [Editing a detector]({{site.url}}{{site.baseurl}}security-analytics/usage/detectors/#editing-a-detector).
```

**Review by Naarcha-AWS — APPROVED** [review]
A couple comments, but looks good.

File: _security-analytics/sec-analytics-config/detectors-config.md [hunk 4]
```diff
@@ -48,9 +50,16 @@ Security Analytics takes advantage of prepackaged Sigma rules for security event
...
 
+### Default field mappings
+
+Once you navigate to the **Configure field mapping** page, the system attempts to automatically map fields between the two sources. The **Default mapped fields** table contains mappings that the system created automatically after defining the detector. As shown in the image that follows, when the field names are similar to one another the system can successfully match the two.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/default-mappings.png" alt="Field mapping example for pending mappings" width="85%">
+
+Although these automatic matches are normally dependable, it's still a good idea to review the mappings and verify that they are correct and matched as expected. If you find a mapping that doesn't appear to be accurate, you can use the dropdown list as described in the [Pending field mappings](#pending-field-mappings) section that follows to correct the field mapping.
```
[the remark this answers is not in the document]
Comment (cwillum) [reply _security-analytics/sec-analytics-config/detectors-config.md:58]: I can repeat that these are mappings in the **Default mapped fields** table.
"Although these automatic matches are normally dependable, it's still a good idea to review the mappings in the **Default mapped fields** table and verify that they are correct and matched as expected."

**Pull request merged**